### ISSUE TITLE
Is it possible to realize some stop condition?

### ISSUE THREAD

**Issue by oradzhabov** [Nicholaswogan/numblsoda#11]
It looks like yours solution is fastest in Python ODE tasks.

In solve_ivp() we have events which allow to implement stop condition during ODE process solving.
Is it possible to find/implement something similar in NumbaLSODA?

It could be useful if I dont know real parameter's range of integration but know where it should be stopped during entire integration process.

Thanks.

**Comment by Nicholaswogan** [comment]
Nope. The code does not have any build-in event finding. It would be possible to implement, but I think it would be a lot of work. The best way might be to re-write `solve_ivp` in C++:  https://github.com/scipy/scipy/blob/master/scipy/integrate/_ivp/ivp.py#L156

**Comment by GenerelSchwerz** [comment]
Not that it means much coming from a random, but I'd love to see support on this. Just food for thought. :+1:
